pkg/bootstrap/render.go

```diff
@@ -137,7 +137,7 @@ func (b *KlusterletManifestsConfig) Generate(ctx context.Context, clientHolder *
 	// Files depends on the install mode
 	var files []string
 	switch b.InstallMode {
-	case operatorv1.InstallModeHosted:
+	case operatorv1.InstallModeHosted, operatorv1.InstallModeSingletonHosted:
```

Comment:
the mode value comes from https://github.com/stolostron/managedcluster-import-controller/blob/f92eb22264afeec6dcd485a313f79dd8733476d4/pkg/helpers/helpers.go#L634 , it seems that it is not possible to be `operatorv1.InstallModeSingletonHosted`

Reply:
hrm I thought we will set the mode in annotation to be hosted. We do not need user to change the annotation to singletonhosted.